db/pre-re/item_db.conf

```diff
@@ -67027,16 +67027,16 @@ item_db: (
 		sitting: true
 	}
 	Script: <"
-		if(rand(1,10)==1) itemskill AL_TELEPORT,1;
-		else if(rand(1,10)==2) itemskill AL_TELEPORT,3;
-		else if(rand(1,10)==3) percentheal 50,0;
-		else if(rand(1,10)==4) percentheal 0,50;
-		else if(rand(1,10)==5) end;
-		else if(rand(1,10)==6) getitem 512,1;
-		else if(rand(1,10)==7) itemskill ALL_REVERSEORCISH,1;
-		else if(rand(1,10)==8) specialeffect2 247;
-		else if(rand(1,10)==9) specialeffect2 338;
-		else specialeffect2 10;
+		if (rand(1,10) == 1) itemskill AL_TELEPORT,1;
+		else if (rand(1,10) == 2) itemskill AL_TELEPORT,3;
+		else if (rand(1,10) == 3) percentheal 50,0;
+		else if (rand(1,10) == 4) percentheal 0,50;
+		else if (rand(1,10) == 5) percentheal 50,50;
+		else if (rand(1,10) == 6) getitem Apple,1;
+		else if (rand(1,10) == 7) itemskill ALL_REVERSEORCISH,1;
+		else if (rand(1,10) == 8) specialeffect2 EF_MAPPILLAR2;
+		else if (rand(1,10) == 9) specialeffect2 EF_ANGEL2;
+		else specialeffect2 EF_COIN;
```

Comment:
It's not this commit's fault, but this script feels wrong (or is it intended?)

The various cases have different chance (because of the separate calls to `rand()`):

```
		if (rand(1,10) == 1) itemskill AL_TELEPORT,1; // 10.00%
		else if (rand(1,10) == 2) itemskill AL_TELEPORT,3; // 9.00%
		else if (rand(1,10) == 3) percentheal 50,0; // 8.10%
		else if (rand(1,10) == 4) percentheal 0,50; // 7.29%
		else if (rand(1,10) == 5) percentheal 50,50; // 6.56%
		else if (rand(1,10) == 6) getitem Apple,1; // 5.91%
		else if (rand(1,10) == 7) itemskill ALL_REVERSEORCISH,1; // 5.31%
		else if (rand(1,10) == 8) specialeffect2 EF_MAPPILLAR2; // 4.78%
		else if (rand(1,10) == 9) specialeffect2 EF_ANGEL2; // 4.31%
		else specialeffect2 EF_COIN; // 38.74%
```